src/net.h

```diff
@@ -226,6 +226,8 @@ class CNodeStats
     TransportProtocolType m_transport_type;
     /** BIP324 session id string in hex, if any. */
     std::string m_session_id;
+    /** whether this peer forced its connection by evicting another */
```

Comment:
8c2026848da910fdebff0a9f73e29f1f6ae81e43

So forced actually means two things to you:
1. Before connecting — something that *might* evict another peer
2. After connecting — something that *has* evicted another peer

I think this is confusing and better to use different words. RPC can expose "force_evicted" flag (to cover the latter case and apply to the `Connection`), while "forceInbound" can stay in the network/potential-connection context (the former case).

Reply:
Apparently, in the previous commit 75868022a904c1f77871abf962bf9b88a9c5faf6 you assign `forced = true;` even for non-`forceInbound` peers. 

So basically if a node had 8 connections which are non-`forceInbound` but still did eviction, a real `forceInbound` connection won't be able to join (see how you count nForced). Is that intended? Seems like a bug to me.